fix(components): components to swap sides (left/right) when theme direction='rtl' (#3837)
chakra-ui/chakra-ui: .changeset/itchy-lions-provide.md

---

> "@chakra-ui/popper": patch
> "@chakra-ui/utils": patch

Reviewer: ```suggestion
"@chakra-ui/popper": minor
```

This should be minor since it's adding new functionality to the hook rather than fixing a bug like the component packages. 

I'm gonna create a new changeset for `@chakra-ui/utils` because it should be more detailed about what new functions were added.

---

> usePoper accepts direction prop now so it handles splacement for rtl languages

Reviewer: ```suggestion
`usePopper` now accepts a `direction` prop so it can handle placement for RTL languages.
```
